Automattic/woocommerce-payments
# Update Outdated Express Checkout Settings Notification

Fixes #9193 

### Changes proposed in this Pull Request









The changes in this PR update a notification in Express Checkout Settings from the `Notification` component to the `InlineNotification` component. These changes were applied to match the latest designs, as detailed in #9193. Since this is a minor change, tests were omitted.

### Testing instructions





**Confirm Issue: Outdated Notification Exists In Express Checkout Settings**
* Checkout to the `develop` branch.
* As a merchant, navigate **Payments > Settings**, scroll down to **Express checkouts** section, click on WooPay's **Customize** button, and scroll down to the **Settings** section.
* Confirm the issue as outlined in #9193.

**Confirm Fix: Updated Notification Exists In Express Checkout Settings**
* Checkout to the `update/9193-outdated-express-checkout-notification` branch.
* As a merchant, navigate **Payments > Settings**, scroll down to **Express checkouts** section, click on WooPay's **Customize** button, and scroll down to the **Settings** section.
* Confirm the issue as outlined in #9193 no longer exists.

-------------------


**Post merge**
